app/assets/javascripts/active_admin/application.js.coffee

```diff
@@ -1,10 +1,19 @@
 # Initializers
 $(document).on 'ready page:load', ->
   # jQuery datepickers (also evaluates dynamically added HTML)
-  $(document).on 'focus', '.datepicker:not(.hasDatepicker)', ->
+  datepicker_selector = 'input.datepicker:not(.hasDatepicker)'
+
+  $(document).on 'focus', datepicker_selector, ->
+    $input = $(@)
+
+    # Only applying datepicker to compatible browsers
+    is_web_kit = /WebKit/.test(navigator.userAgent)
+    is_date_type = $input.attr('type') is 'date'
+    return if is_web_kit and is_date_type
```

Comment:
The test /WebKit/.test(navigator.userAgent) is hardcoded for WebKit.

There is a better solution. Create a input, set the type date and then read the type. If the type is date, the browser supports type date, if the type is text it don't support date.

This should be work:
```coffeescript
return if $input.type is "date"
```

Sorry for the confusing, I have read your code wrong and my example was bad.